client/src/components/common/forms/Signin/Signin.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Signin = () => {
+  return (
+    <div className="relative max-w-md">
+      <h1 className="text-3xl font-bold">Sign Up</h1>
+      <form className="flex flex-wrap -mx-3">
+        <div className="w-full p-3">
+          <input
+            type="text"
+            placeholder="Email"
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
+        <div className="w-full p-3">
+          <input
+            type="password"
+            placeholder="Password"
+            className="w-full border outline-0 border-inputBorder p-2"
+          />
+        </div>
```

Comment:
can we create new Input component, and use that

Reply:
Yes. Working on it. Noted!